app/helpers/interactive_run_helper.rb

```diff
@@ -34,6 +34,13 @@ def interactive_data_div(interactive,run)
     if (run)
       data['puturl'] = put_url(interactive,run)
       data['geturl'] = get_url(interactive,run)
+      if run.collaboration_run
+        collaborator_urls = []
+        run.collaboration_run.collaborators_runs(run.activity, run.user).each do |collaborator_run|
+          collaborator_urls.push(get_url(interactive,collaborator_run))
```

Comment:
I was keeping with the get_url/put_url design which is used inside iFramePhone so it can use relative urls.  These urls are also put in the encoded laraParameters url parameter so duplicating the host and port is going to take up a lot of space.